### ISSUE TITLE
sandpack-cdn-v2.codesandbox.io returns 500 for existing package

### ISSUE THREAD

**Issue by ocavue** [codesandbox/sandpack-cdn#23]
I got the following error when using Sandpack:

```
Failed to fetch "https://sandpack-cdn-v2.codesandbox.io/v2/deps/QGVnb2lzdC90YWlsd2luZGNzcy1pY29uc0AxLjMuMztAaWNvbmlmeS1qc29uL2NpQDEuMS4xMDtlc2J1aWxkLXdhc21AXjAuMTcuMTk7cG9zdGNzc0A4LjQuMjg7cHJvc2VraXRAbGF0ZXN0O3NvbGlkLWpzQF4xLjcuNTt0YWlsd2luZGNzc0AzLjMuMzt0eXBlc2NyaXB0QF41LjAuNDt2aXRlLXBsdWdpbi1zb2xpZEBeMi43LjA7dml0ZUA0LjIuMg==": 
{"status":500,"message":"Package version not found @prosekit/vue@0.0.21","details":"PackageVersionNotFound(\"@prosekit/vue\", \"0.0.21\")"}
```

`@prosekit/vue@0.0.21` was published to NPM 31 hours ago (at 2023-10-27T06:44:47.630Z) so it's weird that sandpack-cdn still can't find it. 

Notice that the URL doesn't require `@prosekit/vue` directly. It requires `prosekit@latest`, and `@prosekit/vue@^0.0.21` is one of the dependencies of `prosekit@latest`.

**Comment by DeMoorJasper** [comment]
Fixed in #27 pending deploy...

**Comment by ocavue** [comment]
Thanks!
